src/ng/compile.js

```diff
@@ -2956,6 +2929,12 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
           (tag !== "img" && (attrNormalizedName === "src" ||
                             attrNormalizedName === "ngSrc"))) {
         return $sce.RESOURCE_URL;
+      } else if (tag === "img" && (attrNormalizedName === "src" ||
```

Comment:
Yup. Security-wise, these three won't execute script, so that doesn't matter too much, but if a user decides to have a more restricted URL sanitization for various reasons, they'd probably like to have the right contexts set in attributes. I'm thinking for instance of webmail apps: if they want to prevent third-party trackers, they could tighten $sce.getTrustedUrl to only accept untrusted urls if they're same-origin.